### PR TITLE
allow for additional params to be passed for staging environments

### PR DESCRIPTION
**Pull request by iamkevingreen**
You can pass additional params to the gtm code so that you can create testing environments and not fire in the production environment of gtm. 

I added a prop that can handle that and test that checks it's contents, open to implementing it differently but think this would be valuable, it also allows you to test gtm without putting the container in live mode.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,8 @@ class GoogleTagManager extends React.Component {
         const gtm = gtmParts({
             id: this.props.gtmId,
             dataLayerName: this.props.dataLayerName || 'dataLayer',
-            additionalEvents: this.props.additionalEvents || {}
+            additionalEvents: this.props.additionalEvents || {},
+            previewVariables: this.props.previewVariables || false
```
Comment (morganjbt) [inline README.md]: You should add a comma at the end of this line to prevent github diff.

Comment (iamkevingreen) [reply README.md]: maybe I haven't had my coffee yet but not sure which line you're talking about? @Morgan-jarry

**Comment by FQ400** [comment]
@iamkevingreen The package wasn't updated to `node8` and `npm5` yet when the `package-lock.json` file was introduced. The file shouldn't be added. I guess it would produce more issues then benefits.

File: README.md (hunk 1 above)
Comment (morganjbt) [reply README.md]: @iamkevingreen I'm talking about the line 49 :).

Comment (morganjbt) [reply README.md]: Simply add `,` add at the end of the line like so :
`previewVariables: this.props.previewVariables || false,`.

**Review by morganjbt — APPROVED** [review]
:shipit:

**Comment by 4min** [comment]
Guys, is there any chance this can be merged soon? Configuring analytics right now and this is exactly the missing feature we need.

**Comment by phildionne** [comment]
Same here, would be great to see this merged :)

**Comment by FQ400** [comment]
ping @gilligan @singles @0xPIT

**Pull request merged**